.github/workflows/build-check.yaml

```diff
@@ -0,0 +1,20 @@
+name: Build Check
+on:
+  pull_request:
+    branches: [ master ]
+#    paths:
+#      - '**.proto'
+
+jobs:
+  docker-example:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
"steps" means they run sequentially, right? would it make more sense to code them as "jobs" to run concurrently?

Reply:
Done.